### PR TITLE
STAC-17885 update documetation

### PR DESCRIPTION
**Pull request by hruhek**
## Accept StackState contribution license agreement
- [x] I hereby acknowledge that my contribution is licensed by the [StackState Open-Source Contribution License](../blob/master/LICENSE.md)

### CODE REVIEW

File: src/main/stackpack/resources/enabled.md [hunk 1]
```diff
@@ -6,19 +6,18 @@ Congratulations! The Splunk StackPack is configured correctly. Data is being rec
...
-
-The Splunk StackPack can be uninstalled by clicking the **UNINSTALL** button. This will remove all Splunk topology specific configuration in StackState.
-
 ## Troubleshooting
 
 Troubleshooting steps for any known issues can be found in the [StackState support Knowledge base](https://l.stackstate.com/ui-splunk-support-kb).
 
+## Uninstall the Splunk StackPack
+
+The Splunk StackPack can be uninstalled by clicking the **UNINSTALL** button. This will remove all Splunk topology specific configuration in StackState.
```
Comment (gggina) [inline src/main/stackpack/resources/enabled.md]: Why `Splunk topology specific configuration` ? is there other splunk configuration that won't be removed when you uninstall?

File: src/main/stackpack/resources/int-splunk-events-overview.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ To configure the Splunk events Agent check:
        * **max_query_chunk_seconds** - Default `3600`.
        * **unique_key_fields** - The fields to use to uniquely identify a record (see below for details). Default `_bkt` and `_cd`.
        * **parameters** - Used in the Splunk API request. The default parameters provided make sure the Splunk saved search query refreshes. Default `force_dispatch: true` and `dispatch.now: true`.
-5. More advanced options can be found in the [example configuration \(github.com\)](https://l.stackstate.com/ui-splunk-events-check-example). 
+5. More advanced options can be found in the [example configuration \(github.com\)](https://l.stackstate.com/ui-splunk-events-v2-check-example). 
 4. Save the configuration file.
 5. Restart StackState Agent V1 to apply the configuration changes.
```
Comment (gggina) [inline src/main/stackpack/resources/int-splunk-events-overview.md]: Agent V2.

Comment (gggina) [reply src/main/stackpack/resources/int-splunk-events-overview.md]: or just `the Agent`

File: src/main/stackpack/resources/int-splunk-metrics-overview.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ To configure the Splunk metrics Agent check:
        * **max_query_chunk_seconds** - Default `3600`
        * **unique_key_fields** - The fields to use to uniquely identify a record (see below for details). Default `_bkt` and `_cd`.
        * **parameters** - Used in the Splunk API request. The default parameters provided make sure the Splunk saved search query refreshes. Default `force_dispatch: true` and `dispatch.now: true`.
-5. More advanced options can be found in the [example configuration \(github.com\)](https://l.stackstate.com/ui-splunk-metrics-check-example). 
+5. More advanced options can be found in the [example configuration \(github.com\)](https://l.stackstate.com/ui-splunk-metrics-v2-check-example). 
 4. Save the configuration file.
 5. Restart StackState Agent V1 to apply the configuration changes.
```
Comment (gggina) [inline src/main/stackpack/resources/int-splunk-metrics-overview.md]: Agent V2 or just `the Agent`

Comment (hruhek) [reply src/main/stackpack/resources/int-splunk-metrics-overview.md]: Fixed

File: src/main/stackpack/resources/int-splunk-events-overview.md (hunk 2 above)
Comment (hruhek) [reply src/main/stackpack/resources/int-splunk-events-overview.md]: Fixed

File: src/main/stackpack/resources/enabled.md (hunk 1 above)
Comment (hruhek) [reply src/main/stackpack/resources/enabled.md]: Fixed

**Review by MelcomVE — APPROVED** [review]

**Pull request merged**